templates/api/api-configmap.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if .Values.api.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: {{ template "opencommerce.name" . }}
+    component: api
+    chart: {{ template "opencommerce.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  name: {{ template "opencommerce.fullname" . }}-api
+  namespace: {{ .Release.Namespace | quote }} 
+data:
+  ## Force UTC timezone
+  TZ: "Etc/UTC"
+  NODE_ENV: "production"
+  ROOT_URL: {{ include "opencommerce.api.url" . | quote }}
+  REACTION_SHOULD_INIT_REPLICA_SET: {{ .Values.api.initReplicaSet | default "false" | quote }}
+  GRAPHQL_PLAYGROUND_ENABLED: {{ .Values.api.enableGraphQlPlayground | default "false" | quote }}
+  GRAPHQL_INTROSPECTION_ENABLED: {{ .Values.api.enableGraphQlIntrospection | default "false" | quote }}
+  STORE_URL: {{ include "opencommerce.web.url" . | quote }}
+  TOKEN_SECRET: {{ .Values.global.tokenSecret}}
+  ## required until rondlite PR is merged.
```

Comment:
Which PR# is this? Should add a reference to it...

Reply:
I'm unable to find an outstanding or recently closed PR from rondlite so have just removed reference to it.